### DISCUSSION TITLE
请问cm311-1sa是否同样刷cm311-1a的包？

### DISCUSSION THREAD

**Discussion by niclau** [ophub/amlogic-s9xxx-armbian#1416] [Amlogic] [answered]
**Device Information | 设备信息**
想购入cm311-1sa

**Armbian Version | 系统版本**
不适用

**Describe the bug | 问题描述**
这机器自带wifi，是否同样刷入s905l3a的包，设备选cm311-1a？
机器自带的wifi可否使用？是否有前人探路了？谢谢

Please provide a detailed description of the issue and, if necessary, attach a screenshot.
不适用

**Comment by ophub** [comment]
可以U盘试试，U盘能用的一般都可以写入emmc

**Comment by niclau** [comment]
主要还没入1SA，
目前手上的1A已经刷入了

**Reply by shendengke** [reply, marked as answer]
我试了CM311-1SA 可以用CM311-1A系统的U盘启动，armbian-install的时候卡死，安装不到emmc中

**Reply by niclau** [reply]
我入手N1了
